docs/features/kubernetes/configuration.md

```diff
@@ -123,6 +123,34 @@ The Google Cloud project to look for Kubernetes clusters in.
 The Google Cloud region to look for Kubernetes clusters in. Defaults to all
 regions.
 
+### `customResources` (optional)
+
+Configures which [custom resources][3] to look for when returning an entity's
+Kubernetes resources.
+
+Defaults to empty array. router.ts Example:
+
+```yaml
+---
```

Comment:
Just a quick thought for this, can we not get the plural form by describing the `CRD` with the API?

Reply:
I think this is a great idea and  was actually recommended by @nora99 also. I think we can add it later as a feature.

Reply:
Cool - let's iterate!